test/index.test.js

```diff
@@ -6,43 +6,42 @@ var CONTEXT = {
   functionVersion: '1',
   invokedFunctionArn: 'arn:aws:lambda:eu-west-1:655240711487:function:LambdaTest:$LATEST'
 };
-
-var EVENT = {
-  'Records': [
-    {
-      'Sns': {
-        'Message': '{\"data\":{\"context\":{\"market\":\"dk\",\"language\":\"en-EN\",\"userId\":\"8aeb3560-0b92-11e6-9605-eb677966096c\"},\"query\":{\"passengers\":[{\"birthday\":\"1986-07-14\"},{\"birthday\":\"1986-07-14\"},{\"birthday\":\"2012-07-14\"},{\"birthday\":\"2015-07-14\"}],\"hotels\":[\"hotel:NE.wvHotelPartId.186356\",\"hotel:NE.wvHotelPartId.197915\",\"hotel:NE.wvHotelPartId.197941\"]}},\"id\":\"12345\"}'
-      }
-    }
-  ]
-};
-
-console.log(' - - - - - - - - - - - - - - - - - - - - TEST SNS MESSAGE:');
-console.log(JSON.stringify(EVENT, null, 2));
-console.log(' - - - - - - - - - - - - - - - - - - - - - - - - - - - - -');
-
-describe('Search request handler ', function () {
```

Comment:
The SNS message in the commented out test is 100% valid but its meant to be simpler with only a handful of hotels being searched for. Flip a coin. Happy to keep or remove on next iteration. 👍